phoenix-core/src/it/java/org/apache/phoenix/end2end/QueryMoreIT.java

```diff
@@ -17,26 +17,7 @@
  */
 package org.apache.phoenix.end2end;
 
-import static org.junit.Assert.assertEquals;
```

Comment:
Please use the code template / formatter from https://phoenix.apache.org/develop.html

Reply:
Done